test/parallel/test-event-emitter-errors.js

```diff
@@ -7,4 +7,4 @@ var EE = new EventEmitter();
 
 assert.throws(function() {
   EE.emit('error', 'Accepts a string');
-}, /Accepts a string/);
+}, /Unhandled "error" event. \(Accepts a string\)/);
```

Comment:
People should not be emitting strings or other things, even though we technically support it. So, in most cases, we'll print the rather unhelpful `[object Object]`, like in your previous test, right?